templates/webhook.bin.erb

```diff
@@ -1,4 +1,4 @@
-#!/opt/puppet/bin/ruby
+#!<%= if @is_pe == true or @is_pe == 'true' then '/opt/puppet/bin' else '/usr/bin' end %>/ruby
```

Comment:
if i have [this discussion](https://github.com/puppetlabs/puppetlabs-concat/pull/270) another time, i'll scream :P